tests/classes/test_hypergraph.py

```diff
@@ -173,3 +173,12 @@ def test_hypergraph_attr(edgelist1):
         H['color']
     H['color'] = 'red'
     assert H['color'] == 'red'
+
+
+def test_members(edgelist1):
```

Comment:
Add a line to test when the node does not exist, and when it's not a member of any edge?
assert H.nodes.membership(0) == ?

H.add_node(0)
assert H.nodes.membership((0) == ?

Reply:
Fixed. Now it returns an XGIError.